### DISCUSSION TITLE
Unread articles counter with Smart Folders

### DISCUSSION THREAD

**Discussion by sherlock440** [ViennaRSS/vienna-rss#1301] [Ideas]
Unfortunately, the unread article counter does not work in Smart Folders — it simply does not. It would be great if it appeared (there are ordinary folders).

**Comment by XertroV** [comment]
The patch required to do this is actually really small. See https://github.com/XertroV/vienna-rss/commits/master

However, I have no idea what side effects you might get. I'd expect unread counters to be off for e.g. folders with both feeds and smart folders.

I also made a release but YMMV and run it at your own risk, etc. I suggest compiling it yourself if you'd like those features.

https://github.com/XertroV/vienna-rss/releases/tag/v%2F3.6.0_beta2_w_sfc
